pkg/networkpath/telemetry/telemetry.go

```diff
@@ -17,6 +17,7 @@ import (
 )
 
 // NetworkPathCollectorType represent the source of the network path data e.g. network_path_integration
+// TODO: DEPRECATED in favour of Path.Origin
```

Comment:
Correct me if I'm wrong -- This seems like a similar case to path_id --> pathtrace_id where we have to keep the old field for backwards compatibility. If so, do we have more details to put in this comment on when we can remove this code? The TODO is a little unclear to me.

Reply:
@AyyLam 
This NetworkPathCollectorType is used to build `collector:x` tag used in Network Path Agents OOTB dashboard.

I think we can just update few dashboard widgets to group by both `collector` and `origin`. That should be enough to have backward compatibility.

I think no customer is using those metrics by their own.

Reply:
Ok, maybe it can help to track usage of collectorType to know when to remove `collector` tag from Network Path Agents OOTB dashboard. But I'm not sure we need to bother too much with this since keeping "group by collector" tag in OOTB dashboard has no/low cost.